src/main/java/core/basesyntax/service/operations/BalanceOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        if (transaction.getOperation() == FruitTransaction.Operation.BALANCE) {

```

Comment:
You should not be using '==' for string comparison. Use equals() method instead for string comparison.

Reply:
ok, done